### PR TITLE
fix endpoints

### PR DESCRIPTION
**Pull request by chr-stian**

### CODE REVIEW

File: galaxykit/client.py [hunk 1]
```diff
@@ -20,6 +20,7 @@
 from . import collections
 from . import roles
 from . import __version__ as VERSION
+from .constants import RBAC_VERSION, EE_ENDPOINTS_CHANGE_VERSION
```
Comment (hendersonreed) [inline galaxykit/client.py:23]: I like this approach, thanks!

**Review by hendersonreed — CHANGES_REQUESTED** [review]
Looks good on the whole!

There's an accidental change I think where you revert a url back to `/_ui/v1/`.

Also, please accept the suggestions from the autoformatting bot.

Looks good though, will be good to have this all in and a new clean version released :)

File: galaxykit/containers.py [hunk 2]
```diff
@@ -46,20 +48,28 @@ def create_container(client, name, upstream_name, registry):
...
+        data = {"groups": existing_groups}
+        return client.put(url, data)
 
 
 def inspect_container_namespace(client, ee_name):
     """
     Inspect a container namepsace
     """
-    url = f"{client.ui_ee_endpoint_prefix}execution-environments/namespaces/{ee_name}/"
+    url = f"_ui/v1/execution-environments/namespaces/{ee_name}/"
```
[the remark this answers is not in the document]
Comment (himdel) [reply galaxykit/containers.py:76]: namespaces weren't moved to the same place as repositories .. so both versions are wrong now (see comment in #80) .. but this one works for older versions :)

**Comment by himdel** [comment]
Works for me :+1: 

We'll probably need more descriptive constants once we have more of them (there have been multiple EE endpoint changes already and more are coming - I suspect you'll want to add a comment with PR numbers to each such constant), but this works for now.

(Also, 4.6.0dev is exactly the same thing as 4.6.x .. I wonder if the constants should go with just `"4.6"` instead.)

**Review by himdel — APPROVED** [review]

**Pull request merged**